xbmc/interfaces/legacy/ListItem.h

```diff
@@ -984,17 +984,18 @@ namespace XBMCAddon
       /// to add extra information, such as 'WatchedCount' for tvshow items
       ///
       /// - **Internal Properties**
-      /// | Key           | Description                                     |
-      /// |--------------:|:------------------------------------------------|
-      /// | inputstream   | string (inputstream.adaptive) - Set the inputstream add-on that will be used to play the item
-      /// | IsPlayable    | string - "true", "false" - Mark the item as playable, **mandatory for playable items**
-      /// | MimeType      | string (application/x-mpegURL) - Set the MimeType of the item before playback
-      /// | ResumeTime    | float (1962.0) - Set the resume point of the item in seconds
-      /// | SpecialSort   | string - "top", "bottom" - The item will remain at the top or bottom of the current list
-      /// | StartOffset   | float (60.0) - Set the offset in seconds at which to start playback of the item
-      /// | StartPercent  | float (15.0) - Set the percentage at which to start playback of the item
-      /// | StationName   | string ("My Station Name") - Used to enforce/override MusicPlayer.StationName infolabel from addons (e.g. in radio addons)
-      /// | TotalTime     | float (7848.0) - Set the total time of the item in seconds
+      /// | Key              | Description                                     |
```

Comment:
Please try to change only the new line to keep history (and git blame) clean here. Please also add a @python_v20 revision tags if possible pointing out this property was added

Reply:
oh, was trying to keep the separators in line with above.
Also, this should be back-ported to 19 as the original PR does cause regression that this resolves?

Reply:
Hum...personally I am fine with it as it doesn't change the API (it's a property) and even restores what used to be the default behaviour, but it is really up to the RM to decide here (@DaveTBlake)

Reply:
I dont really see this as a feature. 
The original PR changed default behaviour (which was needed)

This PR just updates that to allow to keep original behaviour (using the property) 
and to also fix the regression of not allowing the 2nd item to override info set by the 1st item.

Essentially, this should have been part of the original PR if we had foreseen the above issues.

Worse case - maybe we could at least backport the fix without the new property.
But that's just removing functionality that add-ons may yet know they need.

Reply:
The regression fix backport is fine, the property I am not so sure as it is indeed new functionality (up to the RM)

Reply:
new functionality to obtain old functionality :)
I'm happy either way as I don't personally use the old functionality.

I'll push update to remove the spacing I added and add the v20 tag